doc/source/_templates/layout.html

```diff
@@ -1,11 +1,35 @@
 {% extends "!layout.html" %}
 
+{%- block extrahead %}
+<style>
+  .main {
```

Comment:
```suggestion
  .container {
      width: 99%;
  }
  .main {
```
Would be better if we expand the width?

Reply:
> Would be better if we expand the width?

I'm trying it now, but I think because the type is small, we'll lose legibility if the page gets wider.

Reply:
I'm trying it out, but I'm thinking the small type may become unreadable if we make the page wider.

Reply:
Wide looks cooler but is getting hard to read. So maybe a subsequent PR to expand the width and make the type bigger as well.

Reply:
I update it to works for small screens only, would it be okay?

Reply:
> So maybe a subsequent PR to expand the width and make the type bigger as well.

okay